spec/lib/pedicel/base_spec.rb

```diff
@@ -356,117 +355,116 @@ def create_special_certificate(ca_key, ca_certificate, config, oids)
     end
 
     it 'does not err when the chain is good' do
-      expect{Pedicel::Base.verify_x509_chain(params)}.to_not raise_error
+      expect{Pedicel::Base.verify_x509_chain(**params)}.to_not raise_error
```

Comment:
Maybe changing the signature of `verify_x509_chain` so that it takes in a regular positional argument instead of keyword arguments would be better.

You wouldn't have to add `**` at every call site.

Reply:
I disagree. 
`Pedicel::Base.verify_x509_chain` accepts three keyword parameters: `root`, `intermediate` and `leaf`. That makes the code more readable than just passing there one positional argument. There's no control over what are you passing and it's not so obvious what should be passed there unless you add some documentation over the method like YARD.